src/app.tsx

```diff
@@ -189,7 +180,7 @@ export default function Chat() {
         },
         body: JSON.stringify({
           username,
-          adminSecret: adminKey,
+          adminSecret: adminKey.trim() || undefined, // Make admin key optional
```

Comment:
i think it should be `adminKey?.trim()` and i don't think you actually need the `||`, `adminKey?.trim()` i think would default to undefined